app/controllers/pokemons_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PokemonsController < ApplicationController
+
+  def new
+    @pokemon = Pokemon.new
+  end
+
+  def create
+    @pokemon = Pokemon.new
+    @pokemon.name = params[:pokemon][:name]
```

Comment:
try to use strong parameters here, which is this stuff: `params.require(:pokemon).permit( ... )`

Reply:
this will prevent malicious users from passing in extra fields to params that could potentially screw up ur db